MonoGame.Framework/Graphics/Texture.OpenGL.cs

```diff
@@ -37,7 +37,10 @@ private void DeleteGLTexture()
         {
             if (glTexture > 0)
             {
-                GraphicsDevice.DisposeTexture(glTexture);
+                if (GraphicsDevice != null)
+                {
+                    GraphicsDevice.DisposeTexture(glTexture);
```

Comment:
Would this need `BlockOnUIThread` too?  @dellis1972 @cra0zy @Jjagg ?

Reply:
There is no need for BlockOnUIThread, since #5850.
DisposeTexture() adds the handler to _disposeNextFrame list.

Looking at it again, I think that the check for GraphicsDevice might not be needed here, it already checks if(glTexture>0) which has to be the initial (-1).
Let me remove it and see if Mac test halt.

Reply:
OK, everything looks fine.